encfs/SSL_Cipher.cpp

```diff
@@ -505,7 +505,7 @@ CipherKey SSL_Cipher::newRandomKey() {
     compute a 64-bit check value for the data using HMAC.
 */
 static uint64_t _checksum_64(SSLKey *key, const unsigned char *data,
-                             int dataLen, uint64_t *const chainedIV) {
+                             int dataLen, const uint64_t *chainedIV) {
```

Comment:
const uint64_t *const chainedIV ?

Reply:
Any reason why we need a `*const` here ?

Reply:
Just as a security, to be sure not to accidentally modify chainedIV pointer later in the function's code ?
Which is not the case actually.
Thx 👍

Reply:
Yes. More const is more better, right?

Reply:
Your commit does two things:
1) It makes one variable const (the variable chainedIV points to)
2) Makes another non-const (the pointer itself).
I agree with (1) but I don't understand (2).